formspree/templates/static_pages/index.html

```diff
@@ -97,7 +97,7 @@ <h4>What about privacy?</h4>
             <div class="col-1-2">
                 <div class="card">
                     <h4>How much does it cost?</h4>
-                    <p>{{config.SERVICE_NAME}} is free for 1000 submissions per email each month. If you need more, there's a <a href="#plans">{{ config.UPGRADED_PLAN_NAME }} Plan</a>.</p>
+                    <p>{{config.SERVICE_NAME}} is free for {{ config.MONTHLY_SUBMISSIONS_LIMIT }} submissions per email each month. If you need more, there's a <a href="#plans">{{ config.UPGRADED_PLAN_NAME }} Plan</a>.</p>
```

Comment:
`{{config.SERVICE_NAME}} is free for {{ config.MONTHLY_SUBMISSIONS_LIMIT }} submissions per form each month. If you need more, there's a <a href="#plans">{{ config.UPGRADED_PLAN_NAME }} Plan</a>.`